opentracing-api/src/main/java/io/opentracing/ActiveSpanSource.java

```diff
@@ -39,12 +38,18 @@
     ActiveSpan activeSpan();
 
     /**
+     * @see #makeActive(Span, ActiveSpan.Observer)
```

Comment:
ambiguous - is it a shortcut for `makeActive(span, null)` or something else? Or perhaps implementation-dependent.